ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -33,15 +33,15 @@
     private String mAmount;
     private String mMobilePhoneNumber;
     private String mEmail;
-    private String mShippingMethod;
+    private @ThreeDSecureShippingMethod int mShippingMethod;
     private ThreeDSecurePostalAddress mBillingAddress;
-    private @ThreeDSecureVersion String mVersionRequested = VERSION_1;
+    private @ThreeDSecureVersion String mVersionRequested = VERSION_2;
     private @ThreeDSecureAccountType String mAccountType;
     private ThreeDSecureAdditionalInformation mAdditionalInformation;
     private boolean mChallengeRequested = false;
     private boolean mDataOnlyRequested = false;
     private boolean mExemptionRequested = false;
-    private UiCustomization mUiCustomization;
+    private UiCustomization mv2UiCustomization;
```

Comment:
```suggestion
    private UiCustomization mV2UiCustomization;
```
This hardly matters, but it looks like we're capitalizing the `v` in `mV1UiCustomization`.

Reply:
This needed to be changed in a few places 👍